hack/release-tools/changelog.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2018 the Velero contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function join { local IFS="$1"; shift; echo "$*"; }
+
+CHANGELOG_PATH='changelogs/unreleased'
```

Comment:
We don't have a `changelogs` directory for this plugin. Is that something that is missing in the process for this one?

Reply:
Yes, we've not set that up for the repo, mainly because this was experimental until 6mo ago.
I created one as part of the v0.1.2 release. We can use that going forward.

Reply:
Of course, I _just_ reviewed that :woman_facepalming: